examples/alpine/column-resizing-performant/src/index.css

```
@@ -171,11 +179,222 @@ button:disabled {
+}
+
+.compact-input {
+  padding: 0 0.25rem;
+}
+
+.wide-action-button {
+  width: 16rem;
+}
+
+.summary-panel {
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)